request.el

```diff
@@ -905,12 +930,12 @@ Currently it is used only for testing.")
   (append
    (list request-curl "--silent" "--include"
          "--location"
-         ;; FIXME: test automatic decompression
-         "--compressed"
          ;; FIMXE: this way of using cookie might be problem when
          ;;        running multiple requests.
          "--cookie" cookie-jar "--cookie-jar" cookie-jar
          "--write-out" request--curl-write-out-template)
+   (when (plist-get :compression (request--curl-capabilities))
```

Comment:
This should be `(plist-get (request--curl-capabilities) :compression)`

Reply:
Thanks, fixed. Why I didn't notice this :(